acceptance/config/aio/options.rb

```diff
@@ -1,6 +1,7 @@
 {
   :type => 'aio',
   :is_puppetserver => true,
+  :puppetserver_confdir => '/etc/puppetlabs/puppetserver/conf.d',
```

Comment:
If we merge this now, then AIO acceptance will fail running against the *current* nightly puppet-server, until builds with https://tickets.puppetlabs.com/browse/SERVER-387 are promoted to nightly. Can we leave this as `/etc/puppetserver/conf.d` for now? Then when the new puppet-server is promoted, we'll encounter failures on our end telling us to change `puppetserver_confdir`?